xarray/backends/api.py

```diff
@@ -961,7 +966,13 @@ def open_mfdataset(
         raise
 
     combined._file_obj = _MultiFileCloser(file_objs)
-    combined.attrs = datasets[0].attrs
+
+    # read global attributes from the attrs_file or from the first dataset
+    if attrs_file is not None:
+        combined.attrs = datasets[paths.index(attrs_file)].attrs
```

Comment:
since `Path` objects get converted to `str`, we need
```python
        if isinstance(attrs_file, Path):
            attrs_file = str(attrs_file)
```
somewhere to allow passing a `Path`. Could you also add a test for that?

Reply:
Thank you for the review. I did not know about pathlib. I've added a test (aa72572).